doofinder-for-woocommerce/includes/class-data-index.php

```diff
@@ -444,10 +444,12 @@ private function load_languages()
 
 		$languages = $this->language->get_languages();
 
+		$this->languages = [''];
 		if (is_array($languages)) {
-			$this->languages = array_keys($languages);
-		} else {
-			$this->languages = [''];
+			$this->languages = [];
+			foreach ($languages as $key => $value) {
+				$this->languages[] = $value['locale'];
+			}
```

Comment:
No need to get the $key of the entry
```suggestion
			foreach ($languages as $language) {
				$this->languages[] = $language['locale'];
			}
```

Reply:
You are right, thanks!